src/pages/novoProduto.js

```diff
@@ -0,0 +1,288 @@
+  background-color: transparent;
+  ::placeholder {
+    color: #4a2f87;
+  }
+  opacity: 0.3;
+  width: 100%;
+  outline: none;
+  :focus {
+    opacity: 1;
+  }
+  margin-bottom: 50px;
+`;
+
+const ContainerAlturaPagina = styled.div`
+  min-height: 100vh;
+  display: flex;
+  flex-flow: column;
+  justify-content: space-between;
+`;
+
+const ContainerTitulo = styled.div`
+  width: 80vw;
+  margin: auto;
+`;
+const ButtonFlex = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const ButtonLojista = styled.button`
+  text-transform: uppercase;
+  border: 3px solid #e44e6d;
+  border-radius: 7px;
+  opacity: 1;
+  width: 180px;
+  height: 40px;
+  font: var(--unnamed-font-style-normal) normal
+    var(--unnamed-font-weight-normal) 16px/25px Poppins;
+  letter-spacing: var(--unnamed-character-spacing-0);
+  text-align: center;
+  font: normal normal normal 16px/25px Poppins;
+  letter-spacing: 0px;
+  color: #0aaa14;
+  opacity: 1;
+  background-color: white;
+  color: #e44e6d;
+  &:hover {
+    background-color: #e44e6d;
+    color: white;
+    transition: 150ms;
+  }
+  &:hover {
+    cursor: pointer;
+  }
+  margin-bottom: 3%;
+`;
+const Letter = styled.p`
+  color: #4a2f87;
+`;
+const ImgDiv = styled.div`
+  width: 200px;
+  height: 200px;
+  background-color: #e44e6d;
+`;
+const Img = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
+class NovoProduto extends Component {
+  state = {
+    productName: "",
+    prodDescription: "",
+    prodPrice: 0,
+    payMeth: "",
+    store: "",
+    prodImg: [],
+    prodInstallments: 0,
+  };
+
+  handleName = (e) => {
+    this.setState({ productName: e.target.value });
+  };
+
+  handleDescription = (e) => {
+    this.setState({ prodDescription: e.target.value });
+  };
+
+  handlePrice = (e) => {
+    this.setState({ prodPrice: Number(e.target.value) });
+  };
+
+  handlePay = (e) => {
+    this.setState({ payMeth: e.target.value });
+  };
+
+  handleStore = (e) => {
+    this.setState({ store: e.target.value });
+  };
+
+  handleImg = (e) => {
+    this.setState({ prodImg: e.target.value });
+  };
+
+  handleInstallments = (e) => {
+    this.setState({ prodInstallments: Number(e.target.value) });
+  };
+
+  addProduct = () => {
+    var axios = require("axios");
+    var data = JSON.stringify({
+      name: this.state.productName,
+      description: this.state.prodDescription,
+      price: Number(this.state.prodPrice),
+      paymentMethod: this.state.payMeth,
+      category: this.state.store,
+      photos: [this.state.prodImg],
+      installments: Number(this.state.prodInstallments),
+    });
+    console.log(data);
+
+    var config = {
+      method: "post",
+      url:
+        "https://us-central1-labenu-apis.cloudfunctions.net/eloFourThree/products",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      data: data,
+    };
+
+    axios(config)
+      .then(
+        function (response) {
```

Comment:
Nesse caso seria melhor fazer com arrow function para não precisar usar o bind! Desde que o javascript lançou as arrow functions a evita essa sintaxe, que é um pouco mais confusa. 

Outra dica é: você não precisa dar um this.setState para cada propriedade, pode setar todas usando um só:

`this.setState({productName: "", prodDescription: "", prodPrice: "", payMeth: "",  store: "", prodImg: "",  prodInstallments: "" })`